docs/sql/sql.md

```diff
@@ -0,0 +1,440 @@
+# SQL Support
+
+SpacetimeDB supports two subsets of SQL.
+One for ad hoc queries against the database.
+These can be made through the [cli] or the [http] endpoint.
+Another for subscriptions made through the [websocket] api or [sdk].
+
+## Subscriptions
+
+```ebnf
+SELECT projection FROM relation [ WHERE predicate ]
+```
+
+The subscription language is a query language.
+It does not support data manipulation commands.
+It is used to subscribe to a set of rows from a single table.
+
+### SELECT
+
+```ebnf
+SELECT ( '*' | table '.' '*' )
+```
+
+The `SELECT` clause determines the table that is being subscribed to.
+A wildcard projection `*` is allowed when the table is unambiguous.
+A qualified wildcard projection `.*` is needed when there are multiple tables.
+Column projections are not allowed in subscriptions.
+
+### FROM
+
+```ebnf
+FROM table [ [AS] alias ] [ [INNER] JOIN table [ [AS] alias ] ON predicate ]
+```
+
+While you can only subscribe to rows from a single table,
+you may reference two tables in the `FROM` clause using a `JOIN`.
+A `JOIN` selects all combinations of rows from its input tables,
+and `ON` determines which combinations are considered.
+Note that if a `JOIN` is used,
+an index must be defined on the join column of both tables,
+so that it can be evaluated efficiently.
```

Comment:
```suggestion
> IMPORTANT: if a `JOIN` is used,
an index *must* be defined on the join column of both tables,
so that it can be evaluated efficiently.
```

so it  stand out